db/deploy/util_functions/upsert_timestamp_columns.sql

```diff
@@ -0,0 +1,136 @@
+-- Deploy connectivity-intake:util_functions/upsert_timestamp_columns to pg
+-- requires: create_roles
+
+begin;
+
+create or replace function connectivity_intake_public.upsert_timestamp_columns(
+  table_schema_name text,
+  table_name text,
+  add_create boolean default true,
+  add_update boolean default true,
+  add_delete boolean default true,
```

Comment:
We are just renaming `add_delete` to `add_archive` (and `deleted_at/by` to `archived_at/by`) in the CIF project, and you might want to do the same. We found that using "deleted" was quite confusing in terms of nomenclature: "delete" might mean two different things (deleting the record vs setting `deleted_at`), which forced us to use terms such as "hard-delete" and "soft-delete" to make the distinction.

Using "archive" instead fixes that confusion